net/uring/all.go

```diff
@@ -9,3 +9,42 @@ import "runtime"
 func Available() bool {
 	return runtime.GOOS == "linux"
 }
+
+type IORingOp = int
```

Comment:
Let's put this (and all the other capability probing code) in a file called capability_linux.go. Other platforms don't care (doesn't need to be in all.go), and it's a nice, simple, standalone bit of code.

Can we call it just `op`? Then the type is `uring.op`, which seems appropriate. Also, unexpected is better.